### DISCUSSION TITLE
Alternate Email Services for Account Authentication.

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15858] [Ideas]
Hi,

I want to plant a seed of an idea I can only hope will blossom into a Beautiful, Youthful Flower...

Besides a Gmail account, it would be nice to see other Email services added as possible authentication methods to Log-Into Runelite. I think the true spirit of Runelite, a sorta compendium of Decentralized community plugins, should (allow) users the ability to Validate/Log-Into their accounts with other Email services besides the notoriously Data-Hoarding throngs of Google Execs...

I realize it's a long shot and possibly an indifferent idea to most...

You can already use multiple different Email services to create an OSRS account, why not allow for more Email services on Runelite? 

That's my rant,
Thanks!

**Comment by raiyni** [comment]
Duplicate of https://github.com/runelite/runelite/discussions/15482
